6/README.md

```diff
@@ -0,0 +1,275 @@
+- DbGetClassAttributeProperty
+- DbGetClassAttributeProperty2
+- DbGetClassAttributePropertyHist
+- DbPutClassAttributeProperty
+- DbPutClassAttributeProperty2
+
+##### ClassProperty
+- DbDeleteClassProperty
+- DbGetClassProperty
+- DbGetClassPropertyHist
+- DbGetClassPropertyList
+- DbPutClassProperty
+
+##### DeviceAttributeProperty
+- DbDeleteDeviceAttributeProperty
+- DbGetDeviceAttributeProperty
+- DbGetDeviceAttributeProperty2
+- DbGetDeviceAttributePropertyHist
+- DbPutDeviceAttributeProperty
+- DbPutDeviceAttributeProperty2
+- DbDeleteAllDeviceAttributeProperty
+
+##### DeviceProperty
+- DbDeleteDeviceProperty
+- DbGetDeviceProperty
+- DbGetDevicePropertyHist
+- DbGetDevicePropertyList
+- DbPutDeviceProperty
+
+##### Free Object Property
+- DbDeleteProperty
+- DbGetProperty
+- DbGetPropertyHist
+- DbGetPropertyList
+- DbPutProperty
+
+##### ClassPipeProperty
+- DbGetClassPipeProperty
+- DbDeleteClassPipeProperty
+- DbPutClassPipeProperty
+- DbGetClassPipePropertyHist
+
+##### DevicePipeProperty
+- DbGetDevicePipeProperty
+- DbDeleteDevicePipeProperty
+- DbDeleteAllDevicePipeProperty
+- DbPutDevicePipeProperty
+- DbGetDevicePipePropertyHist
+
+
+### Attributes
+The database SHALL provide a means of setting and getting attribute values for a class or device.
+
+The database SHALL implement the following attribute-related commands:
+
+##### ClassAttribute
+- DbDeleteClassAttribute
+- DbGetClassAttributeList
+
+##### DeviceAttribute
+- DbDeleteDeviceAttribute
+- DbGetDeviceAttributeList
+
+
+### Aliases
+The database SHALL provide a means of setting and getting a mapping from alias to device name.
+
+The database SHALL provide a means of setting and getting a mapping from alias to device attribute name.
+
+The database SHALL implement the following attribute-related commands:
+
+##### AttributeAlias
+- DbDeleteAttributeAlias
+- DbGetAttributeAlias
+- DbGetAttributeAliasList
+- DbPutAttributeAlias
+- DbGetAttributeAlias2
+- DbGetAliasAttribute
+
+##### DeviceAlias
+- DbDeleteDeviceAlias
+- DbGetAliasDevice
+- DbGetDeviceAlias
+- DbGetDeviceAliasList
+- DbPutDeviceAlias
+
+### Hosts/Servers
+The database SHALL provide a means of defining hosts and servers.
+
+The database SHALL implement the following commands:
+##### Host
+- DbGetHostList
+- DbGetHostServerList
+- DbGetHostServersInfo
+
+##### Server
+- DBAddServer
+- DbDeleteServer
+- DbGetServerList
+- DbGetServerNameList
+- DbRenameServer
+
+##### ServerInfo 
+- DbDeleteServerInfo
+- DbGetServerInfo
+- DbPutServerInfo
+
+### Pipes
+The database SHALL provide a means of querying and deleting pipes.
+
+The database SHALL implement the following pipe-related commands:
+
+##### ClassPipe 
+- DbDeleteClassPipe
+- DbGetClassPipeList
+
+##### DevicePipe
+- DbDeleteDevicePipe
+- DbGetDevicePipeList
+
+### Free Objects
+
+The database SHALL provide a means of querying Free Objects.
+
+The database SHALL implement this free object-related command:
+- DbGetObjectList
+
+### Exported Devices
```

Comment:
This paragraph is probably the most important. 
What do you think to have it first?

Reply:
I agree with Vince.